tests/components/wmspro/responses.py

```diff
@@ -0,0 +1,243 @@
+"""Example JSON responses for the wmspro tests."""
+
+import json
+
+
+def example_config_test():
```

Comment:
If I mock out the whole library then I would basically a) need to duplicate the library, b) find a way to simulate all the dynamic objects created from the JSON responses and c) wouldn't even be testing the interaction between the platforms and library anymore. I understand the general idea, but don't see the point in having a 2nd implementation basically.

Reply:
Why do you need to duplicate the library?

Reply:
If I mock out all classes of the library, then this is basically a simulated recreation of the library or not?

Reply:
The thing what we do is that we create AsyncMocks where we tell what the library would respond. We usually don't imitate logic (I noticed that you have the `mock_action_call`, we usually just check the mock if it is called like we expect and that's it. In a separate test we test that the logic is updated correctly). I think you will end up with a couple of mocks for the objects you use.

It ties together with mocking internals, we don't want to rely on the inner workings of the library.

I dont have a lot of examples because I am on my other PC